RELEASE_NOTES

```diff
@@ -3,5 +3,14 @@
 This listing shows the versions of the OpenARC package, the date of
 release, and a summary of the changes in that release.
 
+0.1.X		2018/??/??
```

Comment:
This should go inside the 0.1.1 entry since it hasn't been shipped yet.

Reply:
I will change it to `0.1.1`. But leave the date as a placeholder.

Reply:
I see. I have to move it.